engine/engine/content/builtins/render/default.render_script

```diff
@@ -135,6 +137,38 @@ local function create_state()
     return state
 end
 
+local function set_camera_world(state)
+    local camera_components = camera.get_cameras()
+
+    -- This will set the last enabled camera from the stack of camera components
+    if #camera_components > 0 then
+        for i = #camera_components, 1, -1 do
+            if camera.get_enabled(camera_components[i]) then
```

Comment:
New function - we shouldn't render with a  disabled camera.

Otherwise we check each camera from the top of the stack and backwards until we find one, and if we don't we use the "other" method of setting view/projection matrices.